db/organizations.go

```diff
@@ -155,3 +186,43 @@ func (ms *MongoStorage) OrganizationsMembers(address string) ([]User, error) {
 	}
 	return users, nil
 }
+
+// addSubscriptionToOrganization internal method adds the subscription to the organiation with
```

Comment:
the comment does not match with the current method name and implementation

Reply:
Updated comment and function renamed to SetOrganizationSubscritption.